src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Options/SimpleConsoleFormatterOptions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Extensions.Logging.Console;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon.Options
+{
+    internal sealed class SimpleConsoleFormatterOptions
+    {
+#if NET5_0_OR_GREATER
```

Comment:
Because dotnet-monitor is using the 5.0+ Logging infrastructure, we want to make sure this is documented in the schema.json file. The Microsoft.Diagnostics.Monitoring.ConfigurationSchema project is currently only building for netcoreapp3.1, so I think you'll need to update that project net50 so that it gets the Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests that targets .NET 5.

Reply:
Just a bit confused about what is being requested by this comment. I see that the Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests linked under ConfigurationSchema's Projects has a path to the netcoreapp3.1 version (which I'm guessing we would want to instead be the net5.0 version?), but I'm not sure what action I'm supposed to be taking.

Reply:
Change the Microsoft.Diagnostics.Monitoring.ConfigurationSchema project from netcoreapp3.1 to net50 so that it picks up the .NET 5.0 version of Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests so that ColorBehavior is written to the schema.json file.